### ISSUE TITLE
pushpin bottle 404 not found

### ISSUE THREAD

**Issue by jkarneges** [Homebrew/homebrew#37295]
Hey folks,

The pushpin package has bottles that were automatically added to the formula, but the bottle files don't exist on homebrew.bintray.com. Maybe some kind of upload/sync bug with the automatic bottling?

```
$ brew install pushpin
==> Downloading https://homebrew.bintray.com/bottles/pushpin-1.0.0.mavericks.bottle.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "pushpin"
Download failed: https://homebrew.bintray.com/bottles/pushpin-1.0.0.mavericks.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading http://packages.fanout.io/source/pushpin-1.0.0.tar.bz2
...
```

**Comment by DomT4** [comment]
Hmm. Yeah, looks like something's amiss. From the Bintray website: ` Path 'homebrew/bottles/pushpin-1.0.0.mavericks.bottle.tar.gz' was not found. `. @mikemcquaid

**Comment by tdsmith** [comment]
Closed in a710941; sorry, my fault. The problem is that a) bintray deletes uploaded files if we don't do a separate publish step (which brew pull --bottle does) within 24 hours, b) we don't autopublish bottles for new formulas, and c) I let too much time elapse.
